src/main/java/seedu/address/logic/commands/task/EditTaskCommand.java

```diff
@@ -87,9 +87,8 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_TASK_NOT_EDITED);
         }
 
-        model.setTask(taskToEdit, editedTask);
         model.setTaskToRoom(taskToEdit, editedTask, room);
-        model.updateFilteredTaskList(Model.PREDICATE_SHOW_ALL_TASKS);
+        //model.updateFilteredTaskList(Model.PREDICATE_SHOW_ALL_TASKS);
```

Comment:
if this is not needed, it can be removed?

Reply:
Yup, this should be removed.